setup.py

```diff
@@ -55,6 +55,9 @@
         # tchannel deps
         'thriftrw>=0.4,<2',
         'threadloop>=1,<2',
+
+        # tracing deps
+        'opentracing>=1.0rc4'
```

Comment:
I made it `opentracing>=1.0rc4,<2`, as pip doesn't like `>=1' when there hasn't been an official OT release, only RCs.